### PR TITLE
Dashboard column change

### PR DESCRIPTION
**Pull request by matthew-sochor**

### CODE REVIEW

**Comment by arikfr** [comment]
I'll have to see how it looks before merging.

Btw, the comment about changing bootstrap.css from your first commit still relevant?

**Comment by matthew-sochor** [comment]
No, the next commit with container-fluid fixed that issue.  I'll edit the first commit message.  

I think a more elegant solution might be to have the ability to toggle between a wide format and the current tall and skinny format. I really wanted the full width/3 columns because we display some dashboards on TVs and the huge margins are a real drawback.

File: rd_ui/app/scripts/filters.js [hunk 1]
```diff
@@ -49,9 +49,12 @@ angular.module('redash.filters', []).
...
     return function (widgetWidth) {
       if (widgetWidth == 1) {
-        return 6;
+        return 4;
+      } else if (widgetWidth == 2) {
+        return 8;
+      } else {
+        return 12;
       }
-      return 12;
```
Comment (arikfr) [inline rd_ui/app/scripts/filters.js]: While you have support for full width widgets in code, there is no UI for it.

Comment (arikfr) [inline rd_ui/app/scripts/filters.js]: Also, looks like the gridster preview wasn't updated for the new dimensions: 

image

**Comment by arikfr** [comment]
I think that until we do some redesign of the whole interface, we should hide this behind a toggle (can be done with ng-class, and adding a toggle to the dashboard options view). Otherwise it just doesn't look good:

image

(@ranbena, when will you implement the TV Mode? #117 :))

**Comment by arikfr** [comment]
I apologize but I can't merge this at current state. But expect improvements to the grid soon enough...

**Pull request closed without merging**